CHANGES

```diff
@@ -41,6 +41,7 @@ Incompatible changes
 * ``sphinx.ext.viewcode`` disabled on singlehtml builder.
 * Use make-mode of ``sphinx-quickstart`` by default.  To disable this, use
   ``-M`` option
+* `html_translator_class` is now deprecated
```

Comment:
TBH, Not planned. Sphinx-2.0 might be better to remove. Personally, I prefer at 1.6 release.